src/pages/Home/Home.jsx

```diff
@@ -3,12 +3,16 @@ import axios from "axios";
 import InfiniteScroll from "react-infinite-scroll-component";
 import Masonry from "react-masonry-css";
 import Photo from "../../components/Photo/Photo";
```

Comment:
why are you not using absolute imports?

Reply:
https://floreacristian.teachable.com/courses/from-beginner-to-professional-developer1/lectures/31967464